app/lib/actions/katello/pulp_selector.rb

```diff
@@ -2,11 +2,29 @@ module Actions
   module Katello
     class PulpSelector < Actions::Base
       def plan(backend_actions, repository, smart_proxy, *args)
+        return_action_output = args.dig(0, :return_output) || false
+        found_action = PulpSelector.select(backend_actions, repository, smart_proxy)
+        fail "Could not locate an action for type #{backend_type}" unless found_action
+        if return_action_output
```

Comment:
could we just do this by default?  Is there any reason not to?

Reply:
Made changes to move the common o/p propagating to a middleware. 
With the middleware to popagate output, propagating here by default would mean propagating o/p for all other flows create/delete/index etc.. Also an additional run phase for all these actions?

Reply:
It would propagate the output for the pulp_selector for those yes, but i think that's fine.  I can't forsee any issue with it.

Reply:
Also, it wouldn't do an additional run phase, it would just 'wrap' the existing run phases (if there is one).